src/util/heap/space_descriptor.rs

```diff
@@ -23,6 +22,7 @@ const MANTISSA_BITS: usize = 14;
 const BASE_EXPONENT: usize = BITS_IN_INT - MANTISSA_BITS;
 
 /* 64-bit */
+#[cfg(target_pointer_width = "64")]
```

Comment:
Maybe explain why this (and perhaps other) constants relates to the pointer width of the target architecture

Reply:
Done